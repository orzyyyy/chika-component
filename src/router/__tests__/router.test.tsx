import React from 'react';
import { shallow } from 'enzyme';
import 'nino-cli/scripts/setup';
import Router from '..';

describe('Router', () => {
  it('render correctly', () => {
    const wrapper = shallow(<Router />);
    expect(wrapper).toMatchSnapshot();
  });
});
