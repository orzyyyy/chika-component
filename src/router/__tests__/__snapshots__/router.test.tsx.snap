// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Router render correctly 1`] = `
<HashRouter>
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      path="/calendar"
    />
    <Route
      component={[Function]}
      path="/container"
    />
    <Route
      component={[Function]}
      path="/swiper"
    />
    <Route
      component={[Function]}
      path="/tabs"
    />
    <Route
      component={[Function]}
      path="/ripple"
    />
    <Route
      component={[Function]}
      path="/upload"
    />
    <Route
      component={[Function]}
      path="/drawer"
    />
    <Route
      component={[Function]}
      path="/transform"
    />
    <Route
      component={[Function]}
      path="/mapbox"
    />
    <Route
      component={[Function]}
      path="/template"
    />
  </Switch>
</HashRouter>
`;
